pkg/infra/main.go

```diff
@@ -113,6 +114,9 @@ func startGameLoop() {
 		lootActions := discussion.ResolveLootDiscussion(*globalState, agentMap, lootPool, agentMap[globalState.CurrentLeader], globalState.LeaderManifesto, lootTally)
 		globalState = loot.HandleLootAllocation(*globalState, &lootActions, lootPool)
 
+		channelsMap = addCommsChannels()
+		trade.HandleTrade(*globalState, agentMap, channelsMap, 5, 3)
```

Comment:
```suggestion
		trade.HandleTrade(*globalState, agentMap, 5, 3)
```
Given that no extra-P2P messaging can take place anyway is there any point in creating these channels that aren't closed or used?